src/components/organisms/dndBox/DnDBox.tsx

```diff
@@ -4,8 +4,8 @@ import { GeometricalDataTO } from "../../../dataAccess/access/to/GeometricalData
 import { PositionTO } from "../../../dataAccess/access/to/PositionTO";
 import { ASPECT_RATIO, WINDOW_FACTOR } from "../../../DavitConstants";
 import { useCurrentHeight, useCurrentWitdh, useCustomZoomEvent } from "../../../utils/WindowUtil";
-import { createDnDItem } from "./DnDWrapper";
-import { DavitPath, DavitPathProps } from "./svg/DavitPath";
+import { DavitPath, DavitPathProps } from "../../atomic/svg/DavitPath";
+import { createDnDItem } from "./fragments/DnDWrapper";
```

Comment:
weil ich es gerade sehe: Vielleicth sollten wir solche create Funktionen von einer Factory machen lassen. Also DndItemFactory.getDndItem() oder so. Ich glaube das ist verständlich weil es ein bekanntes Pattern ist. Aber wenn dann in einem neuen refactoring Ticket

Reply:
Ticket angelegt 👍